docker/config/test.env

```diff
@@ -10,12 +10,11 @@ STATSD_CLIENT=django_statsd.clients.null
 SENTRY_DSN=
 
 # Boto
-AWS_ACCESS_KEY=something
-AWS_SECRET_ACCESS_KEY=something
 resource.boto.calling_format=boto.s3.connection.SubdomainCallingFormat
 resource.boto.resource_class=socorro.external.boto.connection_context.RegionalS3ConnectionContext
 resource.boto.bucket_name=crashstats-test
-TELEMETRY_BUCKET_NAME=telemetry-test
+destination.telemetry.bucket_name=telemetry-test
+telemetry.bucket_name=telemetry-test
```

Comment:
These two were missing from the test configuration. Currently, the test code hard-codes the bucket name to use and all the S3 stuff is mocked out, so this doesn't affect anything ... yet.